pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -471,7 +471,8 @@ protected void internalDeleteTopic(boolean authoritative) {
         if (topic.isReplicated()) {
             // Delete is disallowed on global topic
             log.error("[{}] Delete topic is forbidden on global namespace {}", clientAppId(), topicName);
-            throw new RestException(Status.FORBIDDEN, "Delete forbidden on global namespace");
+            throw new RestException(Status.FORBIDDEN, "Delete forbidden topic is replicated on clusters " +
```

Comment:
nit: it might be good to keep the error message consistent between exception and the logging statement above.